src/GoogleAnalyticsEventForwarder.js

```diff
@@ -515,7 +521,7 @@
 
                     ga('create', fieldsObject);
 
-                    setContentGroup(customFlags);
+                    applyContentGroups(customFlags);
```

Comment:
Also `fieldsObject` isn't a descriptive name IMO. What purpose does it serve?

Reply:
yea it's a GA term. it's a **terrible** term and i was just trying to be consistent with the term they used. i've renamed it to `gaOptionalParameters`.  once we move to the universal `gtag.js` there will no longer be something called `fieldsObject`, i don't think